package.json

```diff
@@ -31,25 +31,28 @@
     "test": "npm run clean && npm run lint && npm run spec"
   },
   "dependencies": {
-    "asar": "^0.13.0",
-    "async": "^2.1.5",
-    "debug": "^2.6.3",
-    "fs-extra": "^2.1.2",
-    "glob": "^7.1.1",
+    "asar": "^0.14.0",
+    "async": "^2.6.0",
+    "debug": "^3.1.0",
+    "fs-extra": "^4.0.3",
+    "g": "^2.0.1",
+    "glob": "^7.1.2",
     "lodash": "^4.17.4",
     "parse-author": "^2.0.0",
     "temp": "^0.8.3",
-    "word-wrap": "^1.2.1",
-    "yargs": "7.0.2"
+    "word-wrap": "^1.2.3",
+    "yargs": "10.0.3"
   },
   "devDependencies": {
-    "chai": "^3.5.0",
-    "eslint": "^3.18.0",
-    "eslint-config-standard": "^7.1.0",
-    "eslint-plugin-promise": "^3.5.0",
-    "eslint-plugin-standard": "^2.1.1",
-    "mocha": "^3.2.0",
+    "chai": "^4.1.2",
+    "eslint": "^4.13.1",
+    "eslint-config-standard": "^10.2.1",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-node": "^5.2.1",
+    "eslint-plugin-promise": "^3.6.0",
+    "eslint-plugin-standard": "^3.0.1",
+    "mocha": "^4.0.1",
     "retry": "^0.10.1",
-    "rimraf": "^2.6.1"
+    "rimraf": "^2.6.2"
   }
```

Comment:
Add `engines`?

Reply:
That's a good idea, we should do that for `electron-installer-debian` too